ngraph/python/tox.ini

```diff
@@ -15,15 +15,17 @@ setenv =
   LD_LIBRARY_PATH = {env:LD_LIBRARY_PATH:{homedir}/ngraph_dist/lib}
   DYLD_LIBRARY_PATH = {env:DYLD_LIBRARY_PATH:{homedir}/ngraph_dist/lib}
   PYBIND_HEADERS_PATH = {env:PYBIND_HEADERS_PATH:}
-  NGRAPH_BACKEND = {env:NGRAPH_BACKEND:"INTERPRETER"}
+  NGRAPH_BACKEND = {env:NGRAPH_BACKEND:"CPU"}
+  PYTHONPATH = {env:PYTHONPATH}
 commands=
   {envbindir}/python setup.py bdist_wheel
   {envbindir}/pip install --no-index --pre --find-links=dist/ ngraph-core
-  flake8 {posargs:src/ examples/ setup.py}
-  flake8 --ignore=D100,D101,D102,D103,D104,D105,D107,W503 test/  # ignore lack of docs in tests
-  mypy --config-file=tox.ini {posargs:src/ examples/}
-  pytest --backend={env:NGRAPH_BACKEND} {posargs:test/}
-
+  flake8 {posargs:src/ setup.py}
+  flake8 --ignore=D100,D101,D102,D103,D104,D105,D107,W503 tests/  # ignore lack of docs in tests
+  mypy --config-file=tox.ini {posargs:src/}
+  ; pytest --backend={env:NGRAPH_BACKEND} {posargs:tests/}
```

Comment:
Since it is a commented code(isn't it?) remove it, please.
```suggestion
```

Reply:
This is the line we ultimately want to use. I added a comment to explain.